app/models/collection.py

```diff
@@ -30,12 +30,29 @@ class Config:
             }
         }
 
+class CollectionFind(BaseModel):
+    dataSource: str = Field(..., example="mongodb")
+    database: str = Field(..., example="todo")
+    collection: str = Field(..., example="tasks")
+
+    class Config:
+        populate_by_name = True
+        arbitrary_types_allowed = True
+        json_encoders = {ObjectId: str}
+        json_schema_extra = {
+            "example": {
+                "dataSource": "mongodb",
+                "database": "todo",
+                "collection": "tasks"
+            }
+        }
+
 
 class CollectionInsert(BaseModel):
     dataSource: str = Field(..., example="mongodb")
     database: str = Field(..., example="todo")
     collection: str = Field(..., example="tasks")
-    document: dict = Field(..., example={"text": "Do the dishes"})
+    document: list = Field(..., example={"text": "Do the dishes"})
```

Comment:
Can you create a separate class that contains the schema to insert many documents? It feels like we need to provide a single document in a list as well if we are invoking `/insertOne` endpoint.